tests/test_averager2d.py

```diff
@@ -267,8 +274,8 @@ def setUp(self):
 
         # Setup shifts, don't shift the base image
         self.shifts = np.zeros((self.n_img, 2))
-        self.shifts[1:, 0] = 2
-        self.shifts[1:, 1] = 4
+        self.shifts[1:, 0] = 1
```

Comment:
These values were changed because we are searching a much smaller grid  now (since the conversion to a disc search).